VisualizationBase/src/ViewItemManager.cpp

```diff
@@ -121,6 +124,57 @@ void ViewItemManager::removeAllViewItems()
 	currentViewItem_ = nullptr;
 }
 
+ViewItem* ViewItemManager::createIfNotExists(const QString name, QPoint position)
```

Comment:
Hmm, so this function does return the existing view item if any? Perhaps a better name for it would be `createOrGet`. Although, the `position` parameter isn't used in case the item already exists.

So thinking about this, I don't quite like it:
* We already have a `newViewItem()` method and now we introduce a second one that essentially also creates a new view item. Both of them are `public` which essentially obscures the interface quite a bit.
* We should have only one method that creates a new view. This method should be public. Behind the scenes it could try to load the view or make a new one or whatever.
* I'm not sure if we should combine the creation method with a method that just returns an existing view. Why don't we require that creating a new view with a specific name/position should have a unique name/position and make the client of this method check that? What exactly are the use cases of this method?